packages/create-context-state/__tests__/tsconfig.json.3424582592

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Just curious, are all these tsconfig files normal? Specifically the `tsconfig.json.<number>` file suffix, the number on the end looks odd to me.

Reply:
I use `pnpm fix` command to correct typecheck, and it generated these files.  
Should I do something else to fix them?